src/routes/docs/advanced/platform/shortcuts/+page.markdoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: article
+title: Keyboard Shortcuts
+description: Learn to navigate the Appwrite Console efficiently and effectively with your keyboard
+---
+
+The Appwrite Console was designed with a keyboard first approach. Keyboard navigation along with shortcuts provide an effective and efficient way to navigate the console and perform common actions rather quickly without the need to reach out for the mouse. These shortcuts also benefit developers relying solely on keyboards, guaranteeing an inclusive and accessible experience. Developer experience is greatly improved by virtue of the keyboard's ease of use, quick and seamless execution of essential actions and the consistency of shortcut keys.
```

Comment:
This is quite verbose.

Maybe we could just introduce the page in one or two sentences?

I think we could remove the explanation about what shortcuts are why shortcuts are good and improve devex.